Loading Data ....
 83%|██████████████████████████████████████████████████████████████████████████████████████████████████▌                    | 8278/9999 [00:33<00:07, 243.75it/s]
Traceback (most recent call last):
  File "/home/user/Documents/DL/Assignment2/partA/test.py", line 74, in <module>
    if __name__ == '__main__':
        ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/Documents/DL/Assignment2/partA/test.py", line 12, in evaluate_and_visualize
  File "/home/user/Documents/DL/Assignment2/partA/dataset.py", line 63, in load_dataset
    for data, label in tqdm(train_dataset_full):
                       ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/Documents/DL/DL_ENV/lib/python3.12/site-packages/tqdm/std.py", line 1181, in __iter__
    for obj in iterable:
               ^^^^^^^^
  File "/home/user/Documents/DL/DL_ENV/lib/python3.12/site-packages/torchvision/datasets/folder.py", line 245, in __getitem__
    sample = self.loader(path)
             ^^^^^^^^^^^^^^^^^
  File "/home/user/Documents/DL/DL_ENV/lib/python3.12/site-packages/torchvision/datasets/folder.py", line 284, in default_loader
    return pil_loader(path)
           ^^^^^^^^^^^^^^^^
  File "/home/user/Documents/DL/DL_ENV/lib/python3.12/site-packages/torchvision/datasets/folder.py", line 264, in pil_loader
    return img.convert("RGB")
           ^^^^^^^^^^^^^^^^^^
  File "/home/user/Documents/DL/DL_ENV/lib/python3.12/site-packages/PIL/Image.py", line 982, in convert
    self.load()
  File "/home/user/Documents/DL/DL_ENV/lib/python3.12/site-packages/PIL/ImageFile.py", line 389, in load
    n, err_code = decoder.decode(b)
                  ^^^^^^^^^^^^^^^^^
KeyboardInterrupt
